ui/tests/acceptance/saml-auth-method-test.js

```diff
@@ -5,28 +5,30 @@
 
 import { module, test } from 'qunit';
 import { setupApplicationTest } from 'ember-qunit';
-import { click, fillIn, find, visit, waitUntil } from '@ember/test-helpers';
+import { click, fillIn, find, visit, waitUntil, waitFor } from '@ember/test-helpers';
 import { setupMirage } from 'ember-cli-mirage/test-support';
 import { Response } from 'miragejs';
 import { DELAY_IN_MS, windowStub } from 'vault/tests/helpers/oidc-window-stub';
 import { AUTH_FORM } from 'vault/tests/helpers/auth/auth-form-selectors';
 import { GENERAL } from 'vault/tests/helpers/general-selectors';
 
 import { logout } from 'vault/tests/helpers/auth/auth-helpers';
+import { getErrorResponse } from 'vault/tests/helpers/api/error-response';
+import sinon from 'sinon';
 
 module('Acceptance | enterprise saml auth method', function (hooks) {
   setupApplicationTest(hooks);
   setupMirage(hooks);
 
   hooks.beforeEach(async function () {
     this.openStub = windowStub();
-    this.server.put('/auth/saml/sso_service_url', () => ({
+    this.server.post('/auth/saml/sso_service_url', () => ({
```

Comment:
Just curious why this changed to `POST` with the api service? I wonder if the endpoint supports both `PUT` and `POST`?

Reply:
Yep, it does ([docs](https://developer.hashicorp.com/vault/api-docs/auth/saml#create-or-update-configuration)). I guess ember data was using `put` but these tests all failed when I swapped to the API service and then I noticed in the dev console the request had changed to a `post`